catalog/conformance-tests/result-innoblocktech-001.json

```diff
@@ -0,0 +1,13 @@
+{
+  "tested_solution": {
+    "solution_id": "innoblocktech",
+    "version": "1.0"
```

Comment:
Please note that versions must have three digits (following semver), e.g. (please replace with the actual version):
```suggestion
    "version": "1.0.0"
```

Reply:
both have been fixed in the latest commit